site/content/entry/citra-progress-report-2018-q2.md

```diff
@@ -0,0 +1,305 @@
+use soft shadows, this can be ignored relatively safely.
+
+The second point though, because games use it very often, it has to be implemented
+accurately. The naïve way would be to simply convert internal textures to RGBA8
+manually, but this would slow rendering down to a crawl. The smart way would be
+to try and trick the graphics card driver into performing a similar conversion,
+but this exposed a lot of bugs in it, leading to instability.
+
+After some very hard work by [wwylele](https://github.com/wwylele), both problems
+were solved by one stone. OpenGL supports an extension called Image Load/Store,
+which allows shaders to read and write to any texture directly. Using this, he
+created a shader that could accurately implement both soft shadows, and convert
+the depth map from its internal format to RGBA8 very quickly.
+
+{{< sidebyside "image" "/images/entry/citra-progress-report-2018-q2/"
+    "shadow_map_before.png=Before Shadow Mapping"
+    "shadow_map_after.png=After Shadow Mapping" >}}
+
+Unfortunately, because Image Load/Store is an optional extension, not every
+OpenGL 3.3 graphics card will support it. In these cases, Citra will simply ignore
+shadow maps, making the rendering inaccurate but usable. Image Load/Store became
+a mandatory part of OpenGl in version 4.2, so every graphics card that complies
+with OpenGL 4.2 is guaranteed to work correctly.<br>
+**Citra only requires a minimum
+compliance to OpenGL 3.3, but OpenGL 4.2 compliance may lead to more accurate
+rendering.**
+
+## [Implement cubeb audio backend](https://github.com/citra-emu/citra/pull/3776) by [darkf](https://github.com/darkf)
+
+[`cubeb`](https://github.com/kinetiknz/cubeb/) is an audio library written by
+the Mozilla Foundation that's cross-platform, and *fast*. Reports from [Dolphin's
+blog](https://dolphin-emu.org/blog/2017/06/03/dolphin-progress-report-may-2017/#50-3937-add-cubeb-audio-backend-by-ligfx)
+showed that it ran at a *third* of the latency of their OpenAL backend. And now,
+thanks to [darkf](https://github.com/darkf), with help from [MerryMage](https://github.com/MerryMage)
+Citra can now benefit from that same low latency audio that Dolphin and Mozilla
+Firefox have. This will lead to significantly less distortion and stuttering in
+your games. Just remember to enable it in the configuration menu!
+
+## [Volume Slider](https://github.com/citra-emu/citra/pull/3831) by [Fearlesstobi](https://github.com/fearlesstobi)
+
+You can now configure the volume level of Citra directly from the configuration
+window. This allows you to set an individual volume level for Citra without
+having to manually set it in your operating system settings.
+
+{{< figure src="/images/entry/citra-progress-report-2018-q2/volume_slider.png"
+    title="Volume Slider" >}}
+
+## [Discord Rich Presence](https://github.com/citra-emu/citra/pull/3883) by [CaptV0rt3x](https://github.com/CaptV0rt3x)
+
+Rich Presence allows you to leverage the totally overhauled "Now Playing"
```

Comment:
`overhauled` - was this ever overhauled? Something snarky could fit here instead.